### PR TITLE
refactor: gradebook table tests

### PR DESCRIPTION
**Pull request by muselesscreator**
**TL;DR -** 
Cleans up GradebookTable component and adds docstrings and unit tests.

Partial ticket completion
JIRA: [AU-3](https://openedx.atlassian.net/browse/AU-3)

**What changed?**

- [ More in depth breakdown of changes ]
- [ Peripheral things that got changed ]
- [ etc... ]

**Developer Checklist**
- [ ] Test suites passing
- [ ] Documentation and test plan updated, if applicable
- [ ] Received code-owner approving review
- [ ] Bumped version number [package.json](../package.json)

**Testing Instructions**

[ How should a reviewer test this PR? ]

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @edx/masters-devs-gta

### CODE REVIEW

File: src/components/GradesTab/GradebookTable/index.jsx [hunk 1]
```diff
@@ -1,83 +1,66 @@
-/* eslint-disable react/sort-comp, react/button-has-type */
+/* eslint-disable react/sort-comp, react/button-has-type, import/no-named-as-default */
```
Comment (leangseu-edx) [inline src/components/GradesTab/GradebookTable/index.jsx:1]: I don't think you need this.

File: src/components/GradesTab/GradebookTable/LabelReplacements.jsx [hunk 2]
```diff
@@ -8,29 +8,37 @@ import {
...
-        key="left-basic"
-        placement="left"
-        overlay={(<Tooltip id="course-grade-tooltip">{totalGradePercentageMessage}</Tooltip>)}
-      >
-        <div>
-          {TOTAL_COURSE_GRADE_HEADING}
-          <div id="courseGradeTooltipIcon">
-            <Icon className="fa fa-info-circle" screenReaderText={totalGradePercentageMessage} />
-          </div>
+export const totalGradePercentageMessage = 'Total Grade values are always displayed as a percentage.';
```
[the remark this answers is not in the document]
Comment (muselesscreator) [reply src/components/GradesTab/GradebookTable/LabelReplacements.jsx:13]: this should eventually be an exported message, and at that point will be tested as such.

File: src/components/GradesTab/GradebookTable/index.jsx (hunk 1 above)
Comment (muselesscreator) [reply src/components/GradesTab/GradebookTable/index.jsx:1]: Alas, I do because GradeButton import

**Review by nsprenkle — APPROVED** [review]
LGTM. Super quick smoke test for modified components also looks good.
